src/main/java/core/basesyntax/WriteToFile.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.time.LocalDate;
+
+public class WriteToFile {
+    public void csvFileWriter(String fileToSavePath, String updateFromFile) throws IOException {
```

Comment:
FileWriter should not know anything about file wou were read data from. This method should take data for writing as argument.

Reply:
done